src/wp-admin/security.php

```diff
@@ -0,0 +1,188 @@
+	<span style="color: #19177C">$page_title</span>,
+	<span style="color: #19177C">$menu_title</span>,
+	<span style="color: #19177C">$menu_slug</span>,
+	<span style="color: #19177C">$function</span> <span style="color: #666666">=</span> <span style="color: #BA2121">&#39;&#39;</span>
+)
+</pre>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div class="meta-box-sortables ui-sortable">
+					<div class="postbox">
+						<h2><span><?php _e( 'Usage' ); ?></span></h2>
+						<div class="inside">
+							<p><?php _e( 'You will need to change the logic in your plugin to check for <code>add_security_page()</code>. For example:' ); ?></p>
+							<!-- HTML generated using hilite.me -->
+							<div style="background: #f8f8f8; overflow:auto;width:auto;border:solid gray;border-width:.1em .1em .1em .8em;padding:.2em .6em;"><pre style="margin: 0; line-height: 125%"><span style="color: #008000; font-weight: bold">if</span> ( <span style="color: #008000">function_exists</span>( <span style="color: #BA2121">&#39;\add_security_page&#39;</span> ) ) {
+	add_security_page(
+		<span style="color: #BA2121">&#39;My Prodigious Plugin&#39;</span>,
+		<span style="color: #BA2121">&#39;My Menu Title&#39;</span>,
+		<span style="color: #BA2121">&#39;my-prodigious-plugin&#39;</span>,
+		<span style="color: #BA2121">&#39;my-output-function&#39;</span>
+	);
+} <span style="color: #008000; font-weight: bold">else</span> {
+	<span style="color: #408080; font-style: italic">// existing code</span>
+}
+</pre>
+							</div>
+							<p><?php _e( 'You will also need to change the logic on your settings pages, but that is outside the scope of this guide. However, you should remember that the idea is to <strong>move</strong> security-related settings, <strong>not</strong> to <em>duplicate</em> them.' ); ?></p>
+						</div>
+						<h2><span><?php _e( 'Security Plugins' ); ?></span></h2>
+						<div class="inside">
+							<p><?php _e( 'If your plugin has nothing but security-related settings it may be more useful to provide a summary of the current settings, with links to where they can be changed.' ); ?></p>
+						</div>
+					</div>
+				</div>
+			</div>
+			<div id="postbox-container-1" class="postbox-container">
+				<div class="meta-box-sortables">
+					<div class="postbox">
+						<h2><span><?php _e( 'Support and Documentation' ); ?></span></h2>
+						<div class="inside">
+							<p>
+								<ul>
+									<li><a href="https://link.classicpress.net/support/security-page" rel="noopener" target="_blank"><?php _e( 'Security Page forum' ); ?></a></li>
+									<li><a href="https://link.classicpress.net/security-page" rel="noopener" target="_blank"><?php _e( 'Documentation' ); ?></a></li>
+									<li><a href="https://link.classicpress.net/my-prodigious-plugin" rel="noopener" target="_blank"><?php _e( 'Example code - My Prodigious Plugin' ); ?></a></li>
```

Comment:
I think it's probably better to leave this link out, and just link to the example code from the documentation page.

Either way, getting real content on the documentation page is a higher priority, to be done before release.